src/Component/Header/style.scss

```diff
@@ -0,0 +1,214 @@
+@import "../../style/mixin";
+@import "../../style/variables";
+.header {
+    height: 6rem;
+    background-color: $whiteColor;
+    box-shadow: 0 4px 20px rgb(38 38 38 / 20%);
+    position: fixed!important;
+    width: 100%;
+    z-index: 999;
```

Comment:
Bổ sung thêm variables cho z-index nhé
![image](https://user-images.githubusercontent.com/84757695/151699651-56a363c1-56a6-40e6-8fac-8568dff100b2.png)
Ví dụ này là của trang dev.to